src/Edit.c

```diff
@@ -1981,7 +1999,7 @@ BOOL EditSaveFile(
       else
         ZeroMemory(lpData,GlobalSize(lpData));
 
-      if (uCodePage == CP_UTF7 || uCodePage == 54936)
+      if ((uCodePage == CP_UTF7) || (uCodePage == 54936))
```

Comment:
What is the purpose of this check for 54936? There are also a few more such encodings on <https://msdn.microsoft.com/en-us/library/windows/desktop/dd317752(v=vs.85).aspx> :
> In addition to SBCS and DBCS code pages, your applications have available the multibyte character set code pages 52936, 54936, 51949, and 5022x, which use an approach similar to that for a DBCS.